src/NUnitEngine/nunit.engine.api/ITestRunner.cs

```diff
@@ -42,9 +42,9 @@ public interface ITestRunner : IDisposable
         /// </summary>
         /// <returns>An XmlNode representing the loaded package.</returns>
         /// <remarks>
-        /// This method is normally optional, since Explore and Run call
+        /// <para>This method is normally optional, since Explore and Run call
         /// it automatically when necessary. The method is kept in order
-        /// to make it easier to convert older programs that use it.
+        /// to make it easier to convert older programs that use it.</para>
```

Comment:
I believe this one is the result of DOC100, which you disabled?

You likely don't need to use `<para>` elements unless the `<remarks>` element has _multiple_ paragraphs.